### PR TITLE
MM-34475 Emoji 13.0 with new library

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-webapp/pull/8220

Updated:
- Messaging > Work with Messages > Using Emojis > Adding Emojis to messages
   - Added details about selecting skin tone when selecting or typing emojis
   - Removed the Accessing Alternate Skin Tones section
   - Corrected RST formatting so that custom emoji sections now display as intended on the same page

### CODE REVIEW

File: source/help/messaging/emoji.rst [hunk 1]
```diff
@@ -6,109 +6,71 @@ Emojis are small, digital images or icons used to communicate or express concept
...
+**Type an Emoji**
 
-Alternatively, open the emoji autocomplete by typing ``:`` followed by at least two characters of the word describing the emoji. 
+Alternatively, open the emoji autocomplete by typing ``:`` followed by at least two characters of the word describing the emoji. Descriptions can include skin tone details for people-based emojis.
 
 .. image:: ../../images/emojiautocomplete.png
    :alt: Emoji Autocomplete
    :scale: 80
    
 It's also possible to create your own `Custom Emoji <https://docs.mattermost.com/help/settings/custom-emoji.html>`__ if the emoji you want to use doesn't exist.
```
Comment (cwarnermm) [inline source/help/messaging/emoji.rst:31]:
```suggestion
It's also possible to `create your own custom emoji <#custom-emoji>`_ if the emoji you want to use doesn't exist.
```

File: source/help/messaging/emoji.rst [hunk 2]
```diff
@@ -6,109 +6,65 @@ Emojis are small, digital images or icons used to communicate or express concept
...
 -------------------------
 
-**Selecting Emojis**
+Selecting an Emoji
+~~~~~~~~~~~~~~~~~~
 
-Select the smile icon inside the Mattermost message input box to open the Emoji Picker.
+Select the **Smile** icon inside the Mattermost message input box to open the Emoji Picker.
 
 .. image:: ../../images/selectemoji.png
```
Comment (esethna) [inline source/help/messaging/emoji.rst:14]: emoji picker was redesigned, needs new screenshot

**Review by esethna — APPROVED** [review]
Approving if screenshot is updated ^

File: source/help/messaging/emoji.rst (hunk 2 above)
Comment (cwarnermm) [reply source/help/messaging/emoji.rst:14]: Thanks! Updated!

File: source/help/messaging/emoji.rst [hunk 3]
```diff
@@ -6,109 +6,65 @@ Emojis are small, digital images or icons used to communicate or express concept
...
-  5. Choose **Select**, and choose what image to use for the emoji. 
-  6. Choose **Save**.
-  
-  .. image:: ../../images/add_custom_emoji.png
-  
-  Once saved, your emoji will be added to the list of Custom Emoji. 
+Using Custom Emojis
+~~~~~~~~~~~~~~~~~~~
+
+To use your Custom Emoji in a message, simply type ":" followed by your emoji name to bring it up in the emoji autocomplete. Other users on the site will also be able to see your emoji in the autocomplete, and will be able to use it in their own messages.
```
Comment (justinegeffen) [inline source/help/messaging/emoji.rst:58]:
```suggestion
To use your Custom Emoji in a message, simply type ``:`` followed by your emoji name to bring it up in the emoji autocomplete. Other users on the site will also be able to see your emoji in the autocomplete, and will be able to use it in their own messages.
```

**Review by justinegeffen — APPROVED** [review]
Just one correction, otherwise LGTM! Approving to expedite the merge.

**Pull request merged**